src/Hydrators/IssueHydrator.php

```diff
@@ -27,4 +29,35 @@ public static function getIssues(PDO $db, $completedFilter): array|null
         $query->setFetchMode(PDO::FETCH_CLASS, Issue::class);
         return $query->fetchAll();
     }
+    public static function getIssue($db, $issue_id)
+    {
+        $issueQuery = $db->prepare("SELECT `issues`.`id`,`issues`.`title`,`issues`.`description` AS 'summary', DATE_FORMAT(`issues`.`date_created`, '%d/%m/%Y') AS 'date_created', `issues`.`reporter`,`issues`.`department`, `issues`.`completed`,`severities`.`name` AS 'severity'
+                FROM `issues` 
+                LEFT JOIN `severities` ON `issues`.`severity` = `severities`.`id`
+                WHERE `issues`.`id` = :id");
+
+        $issueQuery->execute([
+            'id' => $issue_id
+        ]);
+
+        $issueQuery->setFetchMode(PDO::FETCH_CLASS, IssueDetails::class);
+
+        $issue = $issueQuery->fetch();
+
+        if (!$issue) {
+            return null;
+        }
+
+        $commentsQuery = $db->prepare("SELECT `name`, `comment`, DATE_FORMAT(`date_created`, '%d/%m/%Y %H:%i') AS 'date_created' FROM `comments` WHERE issue_id = :id;");
+        $commentsQuery->execute(['id' => $issue->id]);
+        $commentsQuery->setFetchMode(PDO::FETCH_CLASS, Comment::class);
+        $comments = $commentsQuery->fetchAll();
```

Comment:
A good use for the comment hydrator you removed might have been to provide a `getCommentsOnIssue` method through that hydrator, in case you need that functionality elsewhere.

However you do not currently need to share that functionality with anywhere else, so the way you have done it here is also an acceptable approach. 

In terms of code quality or best practice it is a little strange to be hydrating comments in the issue hydrator. This could be easily missed by other developers expecting comment hydrations to happen in a comment hydrator. A developer unfamiliar with this file might spend time creating their own SQL query for this exact thing later on when they work on a different story.

I'm not going to say you need to change anything now, but this is a way that technical debt happens! Perhaps make a note of it, and change it when you do more work on comments.